src/community/features-autopopulate/features-autopopulate-core/src/test/java/org/geoserver/autopopulate/AutopopulateTransactionCallbackTest.java

```diff
@@ -166,7 +169,7 @@ public void testUpdateTransactionElement() {
 
         listener.beforeTransaction(request);
 
-        verify(element, times(2)).getUpdateProperties();
+        verify(element, times(4)).getUpdateProperties();
```

Comment:
Would it be possible to have a test that would have failed before the changes, and would not fail anymore after the fix?

Reply:
Sure, will do.

Reply:
done